north_america/united_states/colorado/parks/great_sand_dunes.md

```diff
@@ -0,0 +1,24 @@
+# Great Sand Dunes National Park and Preserve
+
+[Great Sand Dunes National Park and Preserve](https://www.nps.gov/grsa/index.htm) is a United States National Park located in South-Central Colorado.
+
+The park contains the tallest dunes in North America, with Star Dune rising over 750 feet from top to bottom.
+
+## Scenic backcountry route
+
+Have a high-clearance 4WD vehicle? Take Medano Pass (9,931ft elevation), a scenic off-road trail that passes through the dunes. The 22 mile road includes soft sand for about 4 miles, 9 creek crossings, and a rocky roadbed near the summit of the pass. 

```

Comment:
got any info on where to rent a high-clearance 4wd nearby or at the closest intl airport? i know hertz usually has some jeeps